storybook/pagesmodel.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <QAbstractListModel>
+
+struct PagesModelItem {
+    QString title;
+    QString category;
+};
+
+class PagesModel : public QAbstractListModel
```

Comment:
I've considered `QFileSystemModel` but there are several problems causing that I decided to plain `QAbstractListModel`:

- `QFileSystemModel` doesn't track files' modifications (modification date can be accessed only via method). So `QFileSystemWatcher` is needed anyways to implement live updates later. Tracking files changes from watcher and additions/removals from the models complicates the solution.
- It won't help with sorting bc the key is category read from the file
- It wont help too much with filtering neither - regex is used to easily extract part of file name, otherwise I would use QDir filters and wildcards.

Reply:
OK, fair enough